### DISCUSSION TITLE
Error obtaining query result: "TABLE ,file,database-plugin FROM [...] current folder loaded instead

### DISCUSSION THREAD

**Discussion by emign** [RafaelGB/obsidian-db-folder#528] [Q&A] [answered]
Hi,
I use db folder with a data view query source.
My folder structure looks like this

```
workspace/
├─ 02 Schule/
│  ├─ Schueler/
│  │  ├─ Schueler1.md
│  │  ├─ Schueler2.md
│  │  ├─ Schueler3.md
├─ 2022_23/
│  ├─ courses/
│  │  ├─ 5E.md
```

In 5E.md I have a DB Folder database with this query as source:
```
FROM "02 Schule/Schueler" WHERE Klasse = "5E"
```

But I get this error:

`Error obtaining query result: "TABLE ,file,database-plugin FROM "02 Schule/Schueler" WHERE Klasse = "5E"", current folder loaded instead`

The query seems to be missing the first column name. How can I fix that?

**Comment by RafaelGB** [comment, marked as answer]
Hi @emign ,

Looks like your table does not have any columns (just File). Atleast 1 column is expected for dataview query construction.

I will open an issue to fix this case. While its resolved please just add another column to your table
